flask_smorest/__init__.py

```diff
@@ -92,8 +92,7 @@ def register_blueprint(self, blp, *, parameters=None, **options):
         Must be called after app is initialized.
         """
         blp_name = options.get("name", blp.name)
-        if hasattr(blp, "config_prefix"):
```

Comment:
No point in checking. Later `blp.register_views_in_doc` is called. And it is only available in smorest's blueprints.